plugins/rollbar_dashboards/app/helpers/rollbar_dashboards/dashboards_helper.rb

```diff
@@ -20,5 +20,16 @@ def rollbar_dashboard_container(item_path, settings)
         }
       )
     end
+
+    def item_link(item_title, item_id, dashboard_setting)
+      account_name, project_name = dashboard_setting.account_and_project_name&.split('/')
+      return item_title if account_name.nil? || project_name.nil?
+
+      # Removes path and handles https://api.rollbar.com cases
+      domain = URI.join(dashboard_setting.base_url, '/').to_s.sub('api.', '')
+
+      item_url = "#{domain}#{account_name}/#{project_name}/items/#{item_id}"
```

Comment:
why split them if you just end up joining them again ? :D

Reply:
You mean account_name and project_name? Splitting them allows me to somewhat verify that the passed in `account_and_project_name` is valid -- that a '/' exists in the input and both were passed in.